server/views/components/image-gallery/index.njk

```diff
@@ -1,12 +1,19 @@
 <div class="image-gallery">
-{% for item in imageGallery.items %}
-  <div class="image-gallery__item">
-    {% component 'figure', {
-      type: 'picture',
-      sources: [{src: item.contentUrl, size: item.width}],
-      caption: item.caption,
-      icon: 'other/picture'
-    } %}
-  </div>
-{% endfor %}
+  {% for item in imageGallery.items %}
+    <div class="image-gallery__item">
+      <figure class="figure image-gallery__figure">
+        <picture class="figure__picture">
```

Comment:
There don’t seem to be any styles for this class, either in this patch or on master.

Quick double-check: is that intentional, or is something missing?